src/modules/function.js

```diff
@@ -74,9 +74,10 @@ function handleFormSubmit() {
   const input = document.getElementById('input-id');
   const todo = new List(input.value, Boolean(), id);
   tasks.push(todo);
-  id += 1;
-  renderList();
+  id = tasks.length;
   saveTasksToLocalStorage();
+  renderList();
+  input.value = '';
 }
```

Comment:
- [ ] Index property of added item still does not match the new array length. The first item added should have an id of 1, the second item 2, and so on, even when an item has been deleted from the list the order should remain unchanged.

**Mismatch in index property and array length:**
![image](https://github.com/noelfoka/to-do-list/assets/12745474/b37ea2d6-69bf-4d30-aa76-6adfcdfceab2)